arviz/plots/backends/bokeh/__init__.py

```diff
@@ -30,5 +31,22 @@ def copy_docstring(lib, function):
     return doc
 
 
+def check_bokeh_version():
```

Comment:
In new PR we can add this into the new method added if that sounds good

Reply:
Sounds ok